src/System.Runtime.CompilerServices.VisualC/ref/System.Runtime.CompilerServices.VisualC.cs

```diff
@@ -70,4 +70,17 @@ public sealed partial class RequiredAttributeAttribute : System.Attribute
         public RequiredAttributeAttribute(System.Type requiredContract) { }
         public System.Type RequiredContract { get { return default(System.Type); } }
     }
+    public static partial class IsPinned
+    {
+    }
+    [System.AttributeUsageAttribute((System.AttributeTargets)(8))]
```

Comment:
This is a limitation of GenAPI which doesn't currently have a pretty-printer for these values in attributes.